### PR TITLE
Add commandline options --info --version #15

### PR DESCRIPTION
**Pull request by crazcalm**
Signed-off-by: Marcus Willock <crazcalm@gmail.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by hemani19** [comment]
@crazcalm, thanks for the contribution but this doesn't work like this. You need to define the flags in `plugins.yaml` file first and handle them as defined in [the plugin.yaml descriptor documentation.](https://v1-10.docs.kubernetes.io/docs/tasks/extend-kubectl/kubectl-plugins/#accessing-runtime-attributes)

You can further check some kubectl plugins on github for reference. For example: [kill-plugin](https://github.com/carolynvs/kubectl-kill-plugin/blob/master/main.go#L25)

**Comment by kreddyj** [comment]
@crazcalm As you said it looks like  kubectl version <=1.10 doesn't support boolean flags.
With your PR we can use `info` and `version` using:
`kubectl plugin purser --version='true'`
`kubectl plugin purser --info='true'`
It need not be true it can be any non-empty string.

Fix `make check` [error](https://travis-ci.org/vmware/purser/builds/448036972?utm_source=github_status&utm_medium=notification) raised in this pr #41. you can refer the travis ci or you can run `make check` locally.
Also add defaultValue in plugin.yaml for the flags using `defValue: "true"` Refer: [this](https://v1-10.docs.kubernetes.io/docs/tasks/extend-kubectl/kubectl-plugins/#accessing-runtime-attributes). It will be helpful when user enters wrong flag. Thanks for your PR we will merge this after you fix `make check` error.

**Comment by crazcalm** [comment]
@kreddyj I did what you asked, but I would not merge this.

Setting default values on the plugin.yaml file for info and version means that, regardless of what flags are passed, info and version will always be non-empty strings. As a result, this plugin becomes unusable.

I would suggest removing the default values in the plugins.yaml file.

If I am wrong, or if I have missed something, please correct me.

**Comment by kreddyj** [comment]
@crazcalm yes what you said is correct. You can remove the defValues. Sorry for the confusion :smile:

**Comment by crazcalm** [comment]
@kreddyj no worries. I am just happy that this PR is finally coming to a close. lol

File: plugin.yaml [hunk 1]
```diff
@@ -2,4 +2,9 @@ name: "purser"
 shortDesc: "Cost Insight integration with kubernetes"
 longDesc: >
   Purser gives cost insights of kubernetes deployments.
-command: purser_plugin $@
+command: "./purser"
```
Comment (kreddyj) [inline plugin.yaml]: let it be `purser_plugin $@`, if we change this we need to change all installation steps too. For now lets keep the command same.

**Comment by kreddyj** [comment]
@gurusreekanthc  kubectl version <=1.10 doesn't support boolean flags. So using string flags.
With this PR we can use info and version using:
kubectl plugin purser --version='true'
kubectl plugin purser --info='true'

**Comment by kreddyj** [comment]
@crazcalm thank you for your PR :smiley:

**Pull request merged**